charts/hail-search/values.yaml

```diff
@@ -54,3 +54,8 @@ tolerations: []
 
 global:
   seqrPlatformDeploy: false
+
+exports:
```

Comment:
Don't we want to export the previously set
```
environment:
  HAIL_SEARCH_DATA_DIR: ...
```

here?  So if a user, in the `seqr-platform` specifies a different directory for this:

```
hail-search:
  environment:
     HAIL_SEARCH_DATA_DIR: ...
```

that change would propagate through to the export?

Reply:
Oh wait... we can't interpolate super well in this file.

Reply:
None of these comments are helpful 🤣 ... but I think we probably need approach this different.  Let's set up time to talk through this!